src/pages/Swap/index.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react'
 import ExchangePage from '../../components/ExchangePage'
+import { isAddress } from '../../utils'
 
-export default function Swap({ initialCurrency }) {
-  return <ExchangePage initialCurrency={initialCurrency} />
+function getQueryParam(windowLocation, name) {
```

Comment:
again, url stuff should prob go in another pr

Reply:
yep all this stuff snuck it - removing it for this PR